lib/global-styles.php

```diff
@@ -80,6 +80,13 @@ function gutenberg_experimental_global_styles_get_from_file( $file_path ) {
 			file_get_contents( $file_path ),
 			true
 		);
+
+		$json_decoding_error = json_last_error_msg();
```

Comment:
Comparing by the error message does not seem very reliable. The error messages may change, or localization may impact it. I think we should use https://www.php.net/manual/en/function.json-last-error.php.

Reply:
Changed it to check the error code at https://github.com/WordPress/gutenberg/pull/22698/commits/3346f246ed2bf65c669961249464c6c5e3df5c87